stubs/grpcio/@tests/test_cases/check_aio_multi_callable.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
+from typing import Any, Protocol, cast
```

Comment:
`Any` here isn't used

Reply:
Ah oops thanks. Small detail though, `pre-commit.ci` would've automatically removed it.
```suggestion
from typing import Protocol, cast
```